salad/cli.py

```diff
@@ -49,17 +47,21 @@ def main(args=sys.argv[1:]):
     parser.add_argument('--timeout',
                         help=("Set the saucelabs' idle-timeout for the job"))
 
+    parser.add_argument('--scenarios', '-s',
+                        help=("Limit to the specified scenarios"))
+
     (parsed_args, leftovers) = parser.parse_known_args()
     world.drivers = [parsed_args.browser]
     world.remote_url = parsed_args.remote_url
-    world.remote_capabilities = {}
 
-    if 'version' in parsed_args:
-        world.remote_capabilities['version'] = parsed_args.version
-
-    if 'platform' in parsed_args:
-        world.remote_capabilities['platform'] = parsed_args.platform
+    # prepare the remote capabilities
+    world.remote_capabilities = {}
+    world.remote_capabilities['version'] = parsed_args.browserversion
+    world.remote_capabilities['platform'] = parsed_args.platform
+    world.remote_capabilities['trustAllSSLCertificates'] = True
+    world.remote_capabilities['acceptSslCerts'] = True
```

Comment:
+1 to this format,

Also, I'd strongly prefer that adding security exceptions (like trusting all SSL certs) is an explicit choice by the user.  We shouldn't be disabling security by default.